packages/nuxt/src/runtime/server/clerkMiddleware.ts

```diff
@@ -98,7 +99,17 @@ export const clerkMiddleware: ClerkMiddleware = (...args: unknown[]) => {
     }
 
     const authObject = requestState.toAuth();
-    event.context.auth = authObject;
+    const authHandler = () => authObject;
```

Comment:
I'm planning to raise another PR after shipping this one, which it'll introduce some parameters to this `authHandler`

That's the whole reason we're doing this change as well, so having a function allow to perform custom logic against the auth object